src/blocks/card.scss

```diff
@@ -0,0 +1,110 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+
+  --main-text-color: #060b35;
+  --secondary-text-color: #616070;
+  --acdc-color: #00acdc;
+  --white-color: #fff;
+
+  padding: 16px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-top: 16px;
+    margin-left: 3px;
```

Comment:
The margin-left property should be avoided. Consider using margin: 0 auto to center an image